input-events/idlharness.html

```diff
@@ -0,0 +1,40 @@
+<title>Input Event IDL tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+
+<pre id="untested_idl">
+interface InputEventInit {
+};
+
+interface UIEvent {
+};
+
+interface StaticRange {
+};
+</pre>
+
+<pre id='idl'>
+[Constructor(DOMString type, optional InputEventInit eventInitDict)]
+interface InputEvent : UIEvent {
```

Comment:
Thanks for the review!

I've added a link to https://w3c.github.io/input-events/#h-interface-inputevent and removed the UIEvent portion. Updated test will pass on both Safari and Chrome.

Should we test https://w3c.github.io/uievents/#events-inputevents inside "uievents/interface/"?